plugins/window_size/macos/Classes/FLEWindowSizePlugin.m

```diff
@@ -70,23 +70,31 @@ - (NSArray *)platformChannelRepresentationForFrame:(NSRect)frame;
 @implementation FLEWindowSizePlugin {
   // The channel used to communicate with Flutter.
   FlutterMethodChannel *_channel;
-  // The view displaying Flutter content.
-  NSView *_flutterView;
+   
+  // A reference to the registrar holding the NSView used by the plugin. Holding a reference
+  // since the view might be nil at the time the plugin is created.
+  id<FlutterPluginRegistrar> _registrar;
+}
+
+/// The view displaying Flutter content.
+- (NSView *)flutterView {
```

Comment:
Nit: declare this up in the private interface section (as a property), and move the declaration comment there.